src/main/java/io/cdap/plugin/hubspot/common/BaseHubspotConfig.java

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.hubspot.common;
+
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Macro;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.etl.api.FailureCollector;
+import io.cdap.plugin.common.ReferencePluginConfig;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import javax.annotation.Nullable;
+
+/**
+ * Provides base configuration for accessing Hubspot API
+ */
+public class BaseHubspotConfig extends ReferencePluginConfig {
+
+  public static final String APP_ID = "appId";
+  public static final String API_KEY = "apiKey";
+  public static final String OBJECT_TYPE = "objectType";
+  public static final String TIME_PERIOD = "timePeriod";
+  public static final String REPORT_TYPE = "reportType";
+  public static final String CALLS_PER_DAY = "callsPerDay";
+  public static final String START_DATE = "startDate";
+  public static final String END_DATE = "endDate";
+  public static final String FILTERS = "filters";
+  @Name(FILTERS)
```

Comment:
i guess, static constants need to be separated with one blank line from class fields